modules/network-observability-enriched-flows.adoc

```diff
@@ -32,8 +32,16 @@ spec:
         topic: netobserv-flows-export   <1>
         tls: 
           enable: false                 <2>
+     - type: IPFIX                      <3>
+       ipfix:
+         targetHost: "ipfix-collector.ipfix.svc.cluster.local"
+         targetPort: 4739
+         transport: tcp or udp          <4>
+
   
 ----
 <1> The Network Observability Operator exports all flows to the configured Kafka topic.
 <2> You can encrypt all communications to and from Kafka with SSL/TLS or mTLS. When enabled, the Kafka CA certificate must be available as a ConfigMap or a Secret, both in the namespace where the `flowlogs-pipeline` processor component is deployed (default: netobserv). It must be referenced with `spec.exporters.tls.caCert`. When using mTLS, client secrets must be available in these namespaces as well (they can be generated for instance using the AMQ Streams User Operator) and referenced with `spec.exporters.tls.userCert`.
-. After configuration, network flows data can be sent to an available output in a JSON format. For more information, see _Network flows format reference_
\ No newline at end of file
+<3> You can export flows to IPFIX instead of or in conjunction with exporting flows to Kafka. 
```

Comment:
Can we have the note on both `type: KAFKA` & `type: IPFIX` ?

Reply:
Yes, added